crates/symbolicator-service/src/types.rs

```diff
@@ -272,13 +290,19 @@ impl Default for Platform {
     }
 }
 
-impl fmt::Display for Platform {
-    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+impl AsRef<str> for Platform {
+    fn as_ref(&self) -> &str {
```

Comment:
A manual `as_str -> &'static str` could potentially simplify a bunch of places that assume the `str` lifetime depends on `Platform`.

Reply:
For `Native/Js/JvmPlatform` that could work, but `Platform` isn't static because of the error variant.